homeassistant/components/vallox/sensor.py

```diff
@@ -90,6 +96,7 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
             state_proxy=state_proxy,
             metric_key="A_CYC_REMAINING_TIME_FOR_FILTER",
             device_class=DEVICE_CLASS_TIMESTAMP,
+            state_class=None,
```

Comment:
Maybe default it to `None`, so you don't need to set it when there is none?

Reply:
done